pkgs/by-name/na/naps2/package.nix

```diff
@@ -26,20 +27,8 @@ buildDotnetModule rec {
 
   executables = [ "naps2" ];
 
-  dotnet-sdk =
-    with dotnetCorePackages;
-    sdk_8_0
-    // {
-      inherit
-        (combinePackages [
-          sdk_8_0
-          sdk_6_0-bin
-        ])
-        packages
-        targetPackages
-        ;
-    };
-  dotnet-runtime = dotnetCorePackages.runtime_8_0;
```

Comment:
It could be safer to specify the dotnet version to avoid breakages when the next version comes out and will be the default:
```nix
  dotnet-sdk = dotnetCorePackages.sdk_8_0;
  dotnet-runtime = dotnetCorePackages.runtime_8_0;
```